gsd-web/Jakefile

```diff
@@ -0,0 +1,9 @@
+let { task, desc } = require('jake');
```

Comment:
I'm not sure where would be the most appropriate place (most like the README?), but we'll want to add a reference to the documentation for Jake for future developers easy reference on how to use it.